src/Parser/Core/Modules/Items/Legion/EyeOfCommand.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { calculateSecondaryStatDefault } from 'common/stats';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatPercentage } from 'common/format';
+
+/**
+ * Eye of Command
+ * Your melee auto attacks increase your Critical Strike by X for 10 sec, stacking up to 10 times. This effect is reset if you auto attack a different target.
+ */
+class EyeOfCommand extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  [SPELLS.LEGIONS_GAZE.id] = {
```

Comment:
How about making this a statically named property instead? Since the buff and its ID don't change, and you don't have any other buffs, I think putting it in an array will only make things slightly more complicated.

Reply:
See my older comment for an even conciser way of calculating the average. In general when you are making giant pre-defined objects like this you need to ask yourself if there's not a way to do it runtime. What would you do if there were 1000 entries instead of 10? Adding properties is also make error-sensitive.

Reply:
I mean with a 1000, I'd obviously do something different, with 10 I thought it was fine.